python_modules/dagster/dagster/_core/definitions/decorators/op_decorator.py

```diff
@@ -40,6 +47,38 @@
 if TYPE_CHECKING:
     from ..op_definition import OpDefinition
 
+CODE_ORIGIN_TAG_NAME = "__code_origin"
+
+
+# global var flag to enable/disable attaching code origin to ops and assets
+CODE_ORIGIN_ENABLED = [True]
+
+
+@contextmanager
+def do_not_attach_code_origin() -> Generator[None, None, None]:
```

Comment:
maybe worth some comment to explain this type hint. (but if we move to pytest.fixture, we probably don't need this generator at all)